roles/deploy_user/tasks/main.yml

```diff
@@ -4,7 +4,8 @@
   become: true
   tags:
     - setup
-    - cdhsetup
+    - cdhsetup  # ?
```

Comment:
What does this tag do?

Reply:
I don't think it does anything! I just searched through all our roles and I don't see it used anywhere else. I'll remove